bin/lib/common.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const extend = require('extend')
+const { cyan, bold } = require('colorette')
+const util = require('util')
+
+module.exports = {}
+module.exports.loadConfig = loadConfig
+
+async function loadConfig (program, options) {
+  let argv = extend({}, options, { version: program.version() })
+  let configFile = argv['configFile'] || './config.json'
+
+  try {
+    const file = await util.promisify(fs.readFile)(configFile)
```

Comment:
Is that the only occurence of `fs`?
If so, would `const { readFile } = require('fs')` work also?

Reply:
Plus `const { promisify } = require('util')`.

Reply:
After Node 8, I'm pretty sure all `fs` methods have a Promise interface natively, so the promisify is not even needed.

Reply:
https://nodejs.org/api/fs.html#fs_fs_promises_api currently has stability: experimental.

Reply:
> currently has stability: experimental.

Ohhh right, I was thinking about `fs-extra`, never mind :)

Reply:
Ok, I'll write away promisify.